packages/shorebird_cli/lib/src/common_arguments.dart

```diff
@@ -99,6 +99,18 @@ The path for a public key .pem file that will be used to validate patch signatur
     name: 'private-key-path',
     description: '''
 The path for a private key .pem file that will be used to sign the patch artifact.
+''',
+  );
+
+  /// An argument that allows the user to specify a file where program symbols
+  /// are stored.
+  static const splitDebugInfoArg = ArgumentDescriber(
+    name: 'split-debug-info',
+    description: '''
+In a release build, this flag reduces application size by storing Dart program symbols in a separate file on the host rather than
+                                                             in the application. The value of the flag should be a directory where program symbol files can be stored for later use. These
+                                                             symbol files contain the information needed to symbolize Dart stack traces. For an app built with this flag, the "flutter
+                                                             symbolize" command with the right program symbol file is required to obtain a human readable stack trace.
```

Comment:
Is there a significance to this level of indentation/line width?

Reply:
ah good call that was just a pasting issue